src/dmd/typesem.d

```diff
@@ -2055,8 +2091,13 @@ extern(C++) Type typeSemantic(Type type, const ref Loc loc, Scope* sc)
                 auto ed = s.isEnumDeclaration();
                 if (mtype.members && ed.members)
                     .error(mtype.loc, "`%s` already has members", mtype.id.toChars());
-                else
+                else if (!ed.members)
+                {
                     ed.members = mtype.members;
+                }
+                else
+                {
+                }
```

Comment:
Anything could to end up here?  Otherwise just remove it.

Reply:
I left that there just to point out that I had not forgotten about that case.